### DISCUSSION TITLE
[Feature] Administrator functions in the mobile application

### DISCUSSION THREAD

**Discussion by Pro100x3mal** [immich-app/immich#3736] [Feature Request]
### The feature

Please add admin functions to the mobile app for ios and android. For example, if you log in as an Administrator, you must have access to the administrator settings as in the web client, such as managing jobs, users etc

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by ryanwwest** [comment]
This would be nice even if it just opens a web view of the administrator controls page, which would be better than nothing.

**Comment by shanecbauman** [comment]
I've been looking into the workaround @ryanwwest  suggested. The immich mobile app already has the ability to open urls in an external browser (e.g. the documentation, github, and licenses links at the bottom of the dialog view when you click on the user's profile picture).

A small problem is that when you launch an external browser to handle the admin settings, the browser must already be authenticated with an immich_access_token cookie. Otherwise when launching the url the admin gets redirected to the login page, where they must re-login via the webbrowser before the admin settings are available.

If the webbrowser already has an immich_access_token cookie for the server and its still valid, then the admin gets redirected to the admin settings with no logins required.

So for the smoothest experience the webrowser must already be authenticated, otherwise the user will have to authenticate in the browser before the server admin settings are shown. If this sounds reasonable I can make a pull request. Open to hearing different opinions about this as well.

**Comment by bo0tzz** [comment]
#1789
